pkg/ts/recipe/StoreCook.ts

```diff
@@ -7,28 +7,19 @@
  */
 
 import {logFactory} from '../utils/log.js';
+import {matches} from '../utils/matching.js';
 import {Runtime} from '../Runtime.js';
+import {StoreSpec, Plan} from './types.js';
```

Comment:
Have this export StoreMeta.

Reply:
core uses StoreMeta, are you asking for a re-export or re-declaration?

Reply:
Re-exporting.